lib/capybara/webkit/node.rb

```diff
@@ -47,6 +47,19 @@ def set(value)
       invoke "set", *[value].flatten
     end
 
+    def send_keys(*keys)
+      invoke("sendKeys", [keys].flatten.map { |key|
```

Comment:
Do we want `[keys].flatten` or just `keys`? We don't support multi-key sequences, so I worry that `flatten` is potentially covering up a bug.